<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addMan</class>
 <widget class="QWidget" name="addMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>304</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>304</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>468</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #5a6068;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>52</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
color: white;
</string>
   </property>
   <property name="text">
    <string>Имя:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameMan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px; 
color: white;
background-color: #3a3e44;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
color: white;
</string>
   </property>
   <property name="text">
    <string>Фамилия:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="familyMan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px; 
color: white;
background-color: #3a3e44;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="otchestvoMan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px; 
color: white;
background-color: #3a3e44;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
color: white;
</string>
   </property>
   <property name="text">
    <string>Отчество: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px; 
color: white;
background-color: #3a3e44;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
